core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1562,6 +1562,7 @@ private boolean rerunWF(String workflowId, String taskId, Map<String, Object> ta
 
         // Get the workflow
         Workflow workflow = executionDAOFacade.getWorkflowById(workflowId, true);
+        updateAndPushParents(workflow, "reran");
```

Comment:
I was following the convention that used for restart and retry, like for an example the same method called in the restartWF: ```updateAndPushParents(workflow, "restarted");```. Since the operation is used here to construct the taskexeclog: ```TaskExecLog log = new TaskExecLog(String.format("Sub workflow %s %s.", workflowIdentifier, operation));```, I feel it's more appropriate to use the past tense of "rerun", i.e. "reran" here.